Update governance-faq.md post HIP 138/141 (#1970)
helium/docs: docs/governance/governance-faq.mdx

slug: /governance/faq
---

## What Is veHNT?

> veHNT represents the voting power of the Helium Network.

Reviewer: ```suggestion
veHNT represents the voting power in the Helium Network.
```

> In the Helium ecosystem, HNT holders can receive veHNT by locking their HNT on-chain for a specified

Reviewer: ```suggestion
In the Helium ecosystem, HNT holders can establish a veHNT position by locking their HNT on-chain for a specified
```

period in exchange for voting rights and voting power in all areas of the Helium Network as defined 
in [HIP-51](https://github.com/helium/HIP/issues/336).

> Additionally, this provides other benefits such as earning HNT tokens, requiring the holder to delegate 
> their veHNT to those subnetworks and additionally regularly voting in Network proposals. 
> After the lock-up period ends, the same amount of HNT is released back to the owner's wallet and is now 
> transferable or tradeable as before.

Reviewer: ```suggestion
Additionally, this provides other benefits such as earning HNT tokens if the holder delegates 
their veHNT to those subnetworks and regularly votes in Network proposals. 
After the lock-up period ends, the same amount of HNT is released back to the owner's wallet and is 
transferable or tradeable as before.
```

## What are the requirements for earning HNT tokens with locked up HNT

People who lock up their HNT can use this staked veHNT to vote on proposals in the Helium Network and 
help the Network make collective decisions influencing the future of the Network. It is a requirement
that owners of locked up tokens vote for at least 2 of the most recent 4 veHNT votes or lose staking 
income, or alternatively they can assign a proxy to vote on their behalf.

Additionally, people who lock up their HNT are required to delegate their veHNT to a subnetwork to 
receive HNT token rewards. Delegating veHNT to a subnetwork is now the sole variable affecting the Network 
Utility Score that decides the split of HNT that gets distributed to the IoT and Mobile token emissions 
recipients as described in [HIP 141.](https://github.com/helium/HIP/blob/main/0141-single-token-governance-and-release-proposals.md)
Delegation of veHNT to a subnetwork earns the same for the staker no matter which subtoken they delegate 
towards.

## Where does my HNT Go if I Stake veHNT?

> If you lock up your HNT for veHNT, your HNT will appear as veHNT and remain entirely in your wallet's

Reviewer: ```suggestion
If you lock up your HNT, you will see a veHNT value and it will remain entirely in your wallet's
```

## Do I Have to Delegate My veHNT to a subnetwork?

No. You can hold your veHNT as is or delegate to a subnetwork at any time.
But you will NOT receive HNT for locking your position if you do not delegate and vote.

:::note

Changing delegation is allowed, but you must be delegated at the end of an Epoch to earn 
rewards for that Epoch. If you undelegate ensure you redelegate to continue earning rewards.
:::

## How Granular Is The veHNT Linear Decrease?

> The veHNT decay countdown starts when initiated from the Wallet app/Modular Governance, either 

Reviewer: ```suggestion
The veHNT decay countdown starts when initiated from the Wallet app/Helium Vote, either 
```

been deleted. It can be recreated by transfering HNT into the wallet or swapping other tokens for any 
amount of HNT.

## Can I Automate Re-Staking?

Yes, a constant rolling lock-up period is available called Constant Lock-up. The decay countdown
start point will need positive action to start the lock-up cooldown period.

> Staking 10 HNT for 4 years will get approx 1000 veHNT (10x100) and this will be a rolling 4 year 
> period and will remain at 1000 veHNT until the decay countdown is activated and this will then 
> decrease linearly over 4 years to 0 veHNT and then the HNT can be claimed back.

## Can I Transfer my veHNT or any of the veDNTs to Someone Else?

No. veHNT and the like are fully non-transferable and represent your voting power in the Network. It
is not a token. It is represented in the wallet by a non-transferable NFT.

## How Do I Buy veHNT 

You cannot buy veHNT or move it between wallets. They are not tradable tokens. veHNT can only be obtained
by staking HNT with a lock-up period. 

veHNT and veDNTs have no price in the larger crypto markets.

## What Is the Network Utility Score?

> The Network Utility score was described in [HIP-51](https://github.com/helium/HIP/issues/336) as the DAO Utility score and was updated with [HIP-88,](https://github.com/helium/HIP/issues/707) and was then reduced in complexity with [HIP 141.](https://github.com/helium/HIP/blob/main/0141-single-token-governance-and-release-proposals.md#migrating-to-a-protocol-score-based-only-on-delegation)

Reviewer: ```suggestion
The Network Utility score was described in [HIP-51](https://github.com/helium/HIP/issues/336) as the DAO Utility score and was updated with [HIP-88](https://github.com/helium/HIP/issues/707), and was then reduced in complexity with [HIP 141](https://github.com/helium/HIP/blob/main/0141-single-token-governance-and-release-proposals.md#migrating-to-a-protocol-score-based-only-on-delegation).
```
